tests/test_single_topology.py

```diff
@@ -2005,3 +2005,42 @@ def test_chiral_bond_breaking_1_core_1_dummy(seed):
 
     # should not raise an assertion
     verify_chiral_validity_of_core(mol_a, mol_b, core, ff)
+
+
+def get_vacuum_system_and_conf(mol_a, mol_b, core, lamb):
+    ff = Forcefield.load_default()
+    st = SingleTopology(mol_a, mol_b, core, ff)
+    conf_a = get_romol_conf(mol_a)
+    conf_b = get_romol_conf(mol_b)
+    conf = st.combine_confs(conf_a, conf_b, lamb)
+    return st.setup_intermediate_state(lamb), conf
+
+
+@pytest.mark.nocuda
+@pytest.mark.nightly(reason="slow")
+def test_hif2a_end_state_symmetry():
+    """
+    Test that end-states are symmetric
+    """
+    with resources.path("timemachine.testsystems.data", "ligands_40.sdf") as path_to_ligand:
+        mols = read_sdf(path_to_ligand)
+
+    for idx, mol_a in enumerate(mols):
+        for mol_b in mols[idx + 1 :]:
+            print("testing", mol_a.GetProp("_Name"), "->", mol_b.GetProp("_Name"))
+            core = atom_mapping.get_cores(mol_a, mol_b, **DEFAULT_ATOM_MAPPING_KWARGS)[0]
+            sys_fwd, conf_fwd = get_vacuum_system_and_conf(mol_a, mol_b, core, 0.0)
+            sys_rev, conf_rev = get_vacuum_system_and_conf(mol_b, mol_a, core[:, ::-1], 1.0)
+
+            box = 100.0 * np.eye(3)
+
+            assert sys_fwd.chiral_atom
+            assert sys_rev.chiral_atom
+            assert sys_fwd.torsion
+            assert sys_rev.torsion
+
+            np.testing.assert_allclose(sys_fwd.bond(conf_fwd, box), sys_rev.bond(conf_rev, box))
+            np.testing.assert_allclose(sys_fwd.angle(conf_fwd, box), sys_rev.angle(conf_rev, box))
+            np.testing.assert_allclose(sys_fwd.nonbonded(conf_fwd, box), sys_rev.nonbonded(conf_rev, box))
+            np.testing.assert_allclose(sys_fwd.chiral_atom(conf_fwd, box), sys_rev.chiral_atom(conf_rev, box))
+            np.testing.assert_allclose(sys_fwd.torsion(conf_fwd, box), sys_rev.torsion(conf_rev, box))
```

Comment:
nit/non-blocking: might be nice to have a minimal failing unit test in addition to the nightly (e.g. that just runs the pair `(338, 43)`)

Reply:
added in [633645d](https://github.com/proteneer/timemachine/pull/1418/commits/633645de753013483849ae38430493e4d7b86dff)